tst/EnergyPlus/unit/AirLoopHVACDOAS.unit.cc

```diff
@@ -4338,6 +4337,21 @@ TEST_F(EnergyPlusFixture, AirLoopHVACDOAS_TestGetDesignDayConditions)
     EXPECT_NEAR(thisDOAS.HeatOutTemp, state->dataWeatherManager->DesDayInput(4).MaxDryBulb, 0.000001);
     EXPECT_NEAR(thisDOAS.HeatOutHumRat, state->dataWeatherManager->DesDayInput(4).HumIndValue, 0.000001);
 
+    AirLoopHVACDOAS::AirLoopDOAS anotherDOAS;
```

Comment:
@Myoldmopar here's a question for unit tests, do these two instances need to be deleted at the end of the unit test?